doc/radosgw/adminops.rst

```diff
@@ -1963,7 +1963,8 @@ Binding libraries
 
 ``Golang``
 
- - `QuentinPerez/go-radosgw`_
+ - `IrekFasikhov/go-rgwadmin`_
+ - `QuentinPerez/go-radosgw`_ (DEPRECATED: Because it uses the package `smartystreets/go-aws-auth`_ )
```

Comment:
@IrekFasikhov i found that the author of this package is responsive. instead of claiming it deprecated, could you file an issue for go-radosgw? so it can leverage another SDK (`aws-sdk-go` for instance) for signing requests ?

Reply:
@tchaikov Yes you are right. I removed that "depricated"